src/main/java/de/siegmar/logbackgelf/SimpleJsonEncoder.java

```diff
@@ -58,6 +60,14 @@ SimpleJsonEncoder appendToJSON(final String key, final Object value) {
             appendKey(key);
             if (value instanceof Number) {
                 sb.append(value.toString());
+            } else if (value instanceof String[]) {
+                final String[] values = (String[]) value;
+                sb.append("[");
+                for (int i = 0; i < values.length; i++) {
+                    final String endChar = i < values.length - 1 ? COMMA : EMPTY;
```

Comment:
Please just use the an iterator (with hasNext()) for that.

Reply:
I just used the standard for because of the index, that is needed to control the endChar (at least in my logic). But I am changing this for.